### ISSUE TITLE
Organizer - organizee relationship feature request

### ISSUE THREAD

**Issue by ryantimjohn** [ryantimjohn/action_network_1on1_tracker#4]
Andre asked if we could add an organizer organizee relationship to the one-to-one tracker so that organizers could make see that their organizees were staying active and meeting with people regularly.

**Comment by smithsa** [comment]
Yes that's possible in our database! Will note that in the schema. Curious if there is something in Action Network that has that feature or is worked into the organization's workflow with the CRM? Basically if we can/will have to log it in Action Network.

**Comment by ryantimjohn** [comment]
There's a custom field in AN that we have called "Organize" but it's just free-text so I'm a little hesitant to use.
